File: Concurs Inot C#/Angajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problema_3_CSharp.me.model
{
    [Serializable]
    public class Angajat : Persoana
    {
        public string Username { get; set; }
        public string Parola { get; set; }
        public Angajat(int id, string nume, int varsta, string username, string parola): base(id, nume, varsta)
        {
            Username = username;
            Parola = parola;
        }

        public override string ToString()
        {
            return base.ToString();
        }
    }
}

File: Angajat.java
package me.model;

import java.io.Serializable;

public class Angajat extends Persoana implements Serializable {
    private String username;
    private String parola;

    public Angajat(Integer id, String nume, int varsta, String username, String parola) {
        super(id, nume, varsta);
        this.username = username;
        this.parola = parola;
    }

    public String getUsername() { return username; }

    public void setUsername(String username) { this.username = username; }

    public String getParola() { return parola; }

    public void setParola(String parola) { this.parola = parola; }
}
